jhdf/src/main/java/io/jhdf/object/datatype/FixedPoint.java

```diff
@@ -87,6 +87,10 @@ public boolean isSigned() {
 		return signed;
 	}
 
+	public void setSigned(boolean sig) {
```

Comment:
The design I have up to now is to favour immutable objects, think I would like to stick to this where possible. We might need to introduce some kind of `DatasetBuilder` though as we would want the ability to specify more options like filters, filter options, chunk size.